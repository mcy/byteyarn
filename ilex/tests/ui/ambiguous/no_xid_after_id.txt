/foo%q /null%q /foo%qua
