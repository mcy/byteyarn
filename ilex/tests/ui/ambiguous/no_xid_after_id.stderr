error: unexpected closing `ua`
 --> ambiguous/no_xid_after_id.txt:1:22
  |
1 | /foo%q /null%q /foo%qua
  |                      ^^ expected to be opened by `--ua`
  |
  = note: reported at: ilex/src/rt/emit2.rs:254:22

error: aborting due to previous error
